afew/commands.py

```diff
@@ -50,7 +50,7 @@
     help='continuously monitor the mailbox for new files'
 )
 action_group.add_argument(
-    '-l', '--learn', action='store_true',
+    '-l', '--learn', default=False,
```

Comment:
You should not drop `action='store_true'` since argparse will expect an argument in the non-default case.